pkg/logs/input/file/position.go

```diff
@@ -12,25 +12,82 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/logs/auditor"
 )
 
+// TailingMode type
+type TailingMode uint8
```

Comment:
If there is no reason to explicitly use a uint, and because you're having a second list of consts for the string representation (w `tailingModeToStringRepresentations`), I would probably have been with
```
// TailingMode type
type TailingMode string
// Tailing Modes
const (
	ForceBeginning TailingMode = "forceBeginning"
	ForceEnd = "forceEnd"
	Beginning = "beginning"
	End = "end"
)
```
We would even be able to directly serialize this in the registry. WDYT?

Reply:
I dit the whole int/string to better handle wrong/non-existing/default mode. I remember searching on enum-like concept in go and I end up doing like they do in seelog (https://github.com/cihub/seelog/blob/master/common_loglevel.go#L27-L81) because I found the way it handles default/unknown value relatively convenient.

Reply:
OK I got the idea 👍